public_html/templates/header.html

```diff
@@ -0,0 +1,14 @@
+<header>
+	<div class="header">
+		<h1 class="header-logo-text"> Thorne & Spindle </h1>
+		<nav class = "navigationBar">
+			<ul class="headernav">
+				<li class="navItem"><a href="/women">Women</a></li>
+				<li class="navItem"><a href="/men">Men</a></li>
+				<li class="navItem"><a href="/about">About Thorne&Spindle</a></li>
```

Comment:
Lets just make this `About` not `About Throne&Spindle` since it seems like an eyesore by being too long